api/interfaces/detailedService.ts

```diff
@@ -1,4 +1,4 @@
-export interface GlobalSearchService {
+export interface DetailedService {
```

Comment:
Can you refactor this and rename "DetailedService to "ServiceDetailsService" please.